pinot-core/src/main/java/org/apache/pinot/core/common/datablock/BaseDataBlock.java

```diff
@@ -211,7 +211,7 @@ public BaseDataBlock(ByteBuffer byteBuffer)
    * @param rowId row ID
    * @param colId column ID
    */
-  protected abstract void positionCursorInFixSizedBuffer(int rowId, int colId);
+  protected abstract int computePositionCursorInFixSizedBuffer(int rowId, int colId);
```

Comment:
(nit) update the jaavadoc to include the return value please ?

Reply:
Suggest renaming it to `getOffsetInFixedBuffer`